journals/apps/journals/models.py

```diff
@@ -144,7 +144,7 @@ def get_discovery_data(self):
         return {
             'uuid': str(self.journal.uuid),
             'partner': self.journal.organization.site.siteconfiguration.discovery_partner_id,
-            'organization': self.journal.organization.name,
+            'organization': {'key': self.journal.organization.name},
```

Comment:
I assumed the 'key' part was automatically inserted when I was reviewing the discovery PR. Is there a reason we're sending a dict instead of just the string?